templates/agenda_manuel_item/documents.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends 'layout-with-agenda-item-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda' %}
+
+{% block title %}{% trans %}Documents{% endtrans %}{% endblock %}
+
+{% block sidebar %}
+    {% include 'sidebar/agenda-manuel-item-sidebar.html.twig' %}
+{% endblock %}
+
+{% block header %}
+    <div class="row mt-1 mb-3">
+        <div class="col">
+            <h1>{% trans %}Documents{% endtrans %}</h1>
+            <p class="text-muted"><small>{% trans %}Upload and remove documents{% endtrans %}</small></p>
+        </div>
+        <div class="col-auto">
+            <a name="" id="" class="btn btn-success {% if agenda.isFinished() %} disabled {% endif %}" href="{{ url('agenda_manuel_item_upload_document', {id: agenda.id, agenda_item_id: agenda_item.id}) }}" role="button">{% trans %}Upload document{% endtrans %}</a>
```

Comment:
`name` and `id` attributes could probably be omitted.

Reply:
Yes, removed.